plugins/database/src/db_plugin.cpp

```diff
@@ -145,47 +147,64 @@ namespace
 
             for (auto result = nanodbc::execute(stmt); result.next();) {
                 const auto option_name = result.get<std::string>(0);
-
-                if (option_name == irods::KW_CFG_PAM_PASSWORD_MIN_TIME) {
-                    _out.password_min_time = std::stoll(result.get<std::string>(1));
-                    if (_out.password_min_time < 0) {
-                        return ERROR(CONFIGURATION_ERROR,
-                                     fmt::format("Invalid configuration for option [{}] in namespace [{}]: [{}]",
-                                                 option_name,
-                                                 _namespace,
-                                                 _out.password_min_time));
+                const auto option_value = result.get<std::string>(1);
+
+                if (option_name == irods::KW_CFG_PAM_PASSWORD_MIN_TIME ||
+                    option_name == irods::KW_CFG_PAM_PASSWORD_MAX_TIME) {
+                    const bool min = (option_name == irods::KW_CFG_PAM_PASSWORD_MIN_TIME);
```

Comment:
Consider renaming `min`.

Perhaps something like ...
- `is_min_option`
- `is_min_option_name`
- `is_min`
- `found_min_option`

I don't have strong feelings on this. Up to you.

Reply:
Done